package.json

```diff
@@ -5,10 +5,10 @@
   "license": "CC0-1.0",
   "scripts": {
     "build": "npx @11ty/eleventy",
-    "federalist": "npm-run-all init build uswds:buildSass",
-    "init": "npm-run-all uswds:buildDist uswds:copyAssets",
+    "federalist": "npm run init && npm run build && npm run uswds:buildSass",
+    "init": "npm run uswds:buildDist && npm run uswds:copyAssets",
     "serve": "npx @11ty/eleventy --serve",
-    "start": "npm-run-all -p serve watch",
+    "start": "npm run serve && npm run watch",
```

Comment:
**issue**: The `&&` make the tasks run sequentially. In this case we need them to run in parallel.

We could do this with the following:

```suggestion
    "start": "npm run serve & npm run watch",
```

The only caveat here is, I believe, this won't work in windows environments. So we'd have to use a tool like [concurrently - npm](https://www.npmjs.com/package/concurrently) that works everywhere.

**Steps to reproduce**
1. Run `npm install && npm start`
2. Open `src/_styles/_uswds-theme-custom-styles.scss`
3. Change the body background, i.e.
    ```scss
    // _uswds-theme-custom-styles.scss
    @use "uswds-core" as *;
    
    body {
      background-color: color("red-40v");
    }
    
    ```
4. SASS won't get compiled and nothing changes on refresh.

Reply:
@mejiaj  thanks for flagging this!

I checked out `concurrently` but ran into errors when running the federalist command. The concurrent scripts would exit with code `0` and `1` respectfully.

I did find this alternative solution for windows machines to prefix the command with `start`:

> From windows cmd you can use [start](http://ss64.com/nt/start.html):
>
> `"dev": "start npm run start-watch && start npm run wp-server"`
> Every command launched this way starts in its own window. [^1]

[^1]: https://stackoverflow.com/a/36275359

Since you have a windows machine, could you try running `start npm start` and see if that works? If so, I'll update to use the single `&`.

If not, I'll continue troubleshooting concurrently

Reply:
@mahoneycm my comment on parallel tasks (single `&`) was only for the `start` script. This script needs to run the local server, but also watch and compile changes.

The other scripts being sequential are fine since that's how they're expected to run.

---

Ideally, all platforms would run the same script to get a local version working. It could be tedious and time consuming to maintain a separate version of guidance when a dependency could ensure consistency.

Reply:
@mejiaj thank you for clarifying! I was able to successfully use `concurrently` to allow the scripts to run in parallel. The package didn't bring any dependency vulnerabilities and is working as expected.

Would you mind testing on windows to ensure it's working as expected?